src/component/pages/mainPage/mainPage.tsx

```diff
@@ -0,0 +1,15 @@
+import { FC } from "react";
+import { Navbar } from "../../organism/navbar/navbar";
+import { NavbarMenuProvider } from "../../../context/navbarMenuContext";
+import BoxSliderMenu from "../../organism/boxSliderMenu/boxSliderMenu";
+
+export const MainPage: FC = () => {
+  return (
+    <>
```

Comment:
You don't need to use fragment here as you don't have more elements that the NavbarMenuProvider...